pkg/auth/csp/tanzu.go

```diff
@@ -429,8 +439,40 @@ func GetOrgNameFromOrgID(orgID, accessToken, issuer string) (string, error) {
 	return org.Name, nil
 }
 
-// GetTanzuHubEndpointForTAP retrieves Tanzu Hub Endpoint For TAP SaaS through the CSP API
+// GetTanzuHubEndpointForTAP retrieves Tanzu Hub Endpoint For TAP SaaS through the CSP API or through Central Config as fallback
 func GetTanzuHubEndpointForTAP(orgID, accessToken string, useStagingIssuer bool) (string, error) {
```

Comment:
preexisting, nit:
Is it reasonable to remove "For TAP SaaS" in command and rename function to just GetTanzuHubEndpoint, especially now that we have another means to provide the hub endpoints via CC?